### PR TITLE
[14.0][mig]-account_move_category

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**

### CODE REVIEW

File: account_move_category/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline account_move_category/__init__.py]: I know it is a pain, but can you remove this?

File: account_move_category/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline account_move_category/models/__init__.py]:
```suggestion
# Copyright 2018-23 ForgeFlow S.L.
```

**Review by AaronHForgeFlow — COMMENTED** [review]
Looks good. Minor changes needed. And also rebase is needed.

**Comment by JuanyDForgeflow** [comment]
@AaronHForgeFlow rebase and changes done

**Review by AaronHForgeFlow — APPROVED** [review]
Is good. Can you please remove the copyright from __ini__.py files and  update the copyright date to reflect  "Copyright 2018-23" in all files?

**Pull request merged**